linux-kernel-lemote: update to 5.4.285

KexyBiscuit opened the pull request:
Topic Description
-----------------

- linux-kernel-lemote: fix SRCS URL
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel-lemote: update to 5.4.285
    - NO routine patch due diligence for lemote flavor releases as of now.
    - Track patches at https://github.com/AOSC-Tracking/linux @ aosc/lemote/v5.4.285, current HEAD is d24618de8c547f05d3633d56e299b621181ab5fc.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux+kernel+lemote: bump REL to remove RC markers
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel-lemote: update to 5.4.285-rc1b
    - NO routine patch due diligence for lemote flavor releases as of now.
    - Update stable/linux-stable-rc.git HEAD to 94f2986968b02f873d37da50367b9345f101c922.
    - Track patches at https://github.com/AOSC-Tracking/linux @ aosc/lemote/v5.4.285-rc1, current HEAD is d3537ea091c63ab6cdf4587615d6ddd04dfb43ec.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel-lemote: revert config changes in 0a9b055233a5f380bfd6dadd5f515dc7960ec3d3
- linux-kernel-lemote: sync strip-modules from kernel-build-common
- linux+kernel+lemote: update to 5.4.285-rc1
- linux-kernel-lemote: update to 5.4.285-rc1
    - NO routine patch due diligence for now.
    - Track patches at https://github.com/AOSC-Tracking/linux @ aosc/lemote/v5.4.285-rc1, current HEAD is a95cf72f54584dcc729571901dc6cb355803574c.

Package(s) Affected
-------------------

- linux+kernel+lemote: 5.4.285-1
- linux-kernel-lemote-${__VER}: 5.4.285-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit linux-kernel-lemote linux+kernel+lemote
```

Test Build(s) Done
------------------

**Secondary Architectures**

- [ ] Loongson 3 `loongson3`

KexyBiscuit committed:
linux-kernel-lemote: update to 5.4.285-rc1

- NO routine patch due diligence for now.
- Track patches at https://github.com/AOSC-Tracking/linux @ aosc/lemote/v5.4.285-rc1, current HEAD is a95cf72f54584dcc729571901dc6cb355803574c.

KexyBiscuit committed:
linux+kernel+lemote: update to 5.4.285-rc1

KexyBiscuit committed:
linux-kernel-lemote: sync strip-modules from kernel-build-common

KexyBiscuit committed:
linux-kernel-lemote: revert config changes in 0a9b055233a5f380bfd6dadd5f515dc7960ec3d3

KexyBiscuit commented:
Smoke testing passed on LX-2510.

![image](https://github.com/user-attachments/assets/9abc853c-ad42-4c86-a495-e66cb814fa81)

ouankou commented:
Tested on 3B4000 (A1906).
Before upgrading the kernel:
```bash
(ssh)ouankou@faruzan [ ~ ] $ uname -a
Linux faruzan 5.4.259-aosc-lemote #1 SMP PREEMPT Mon Oct 30 02:53:03 UTC 2023 mips64 GNU/Linux
(ssh)ouankou@faruzan [ ~ ] $ buildbot-benchmark
...
[3047/3047] Linking CXX static library lib/libLLVMXRay.a

real    72m1.332s
user    553m18.122s
sys     17m43.060s
```
After upgrading the kernel:
```bash
(ssh)ouankou@faruzan [ ~ ] $ uname -a
Linux faruzan 5.4.285-aosc-lemote #1 SMP PREEMPT Sat Sep 21 01:02:42 UTC 2024 mips64 GNU/Linux
(ssh)ouankou@faruzan [ ~ ] $ buildbot-benchmark
...
...
[3047/3047] Linking CXX static library lib/libLLVMXRay.a

real    72m21.598s
user    554m57.245s
sys     18m25.028s
```

KexyBiscuit commented:
> Tested on 3B4000 (A1906). Before upgrading the kernel:
> 
> ```shell
> (ssh)ouankou@faruzan [ ~ ] $ uname -a
> Linux faruzan 5.4.259-aosc-lemote #1 SMP PREEMPT Mon Oct 30 02:53:03 UTC 2023 mips64 GNU/Linux
> (ssh)ouankou@faruzan [ ~ ] $ buildbot-benchmark
> ...
> [3047/3047] Linking CXX static library lib/libLLVMXRay.a
> 
> real    72m1.332s
> user    553m18.122s
> sys     17m43.060s
> ```
> 
> After upgrading the kernel:
> 
> ```shell
> (ssh)ouankou@faruzan [ ~ ] $ uname -a
> Linux faruzan 5.4.285-aosc-lemote #1 SMP PREEMPT Sat Sep 21 01:02:42 UTC 2024 mips64 GNU/Linux
> (ssh)ouankou@faruzan [ ~ ] $ buildbot-benchmark
> ...
> ...
> [3047/3047] Linking CXX static library lib/libLLVMXRay.a
> 
> real    72m21.598s
> user    554m57.245s
> sys     18m25.028s
> ```

Thank you for the testing!

KexyBiscuit committed:
linux-kernel-lemote: update to 5.4.285-rc1b

- NO routine patch due diligence for lemote flavor releases as of now.
- Update stable/linux-stable-rc.git HEAD to 94f2986968b02f873d37da50367b9345f101c922.
- Track patches at https://github.com/AOSC-Tracking/linux @ aosc/lemote/v5.4.285-rc1, current HEAD is d3537ea091c63ab6cdf4587615d6ddd04dfb43ec.

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>

KexyBiscuit committed:
linux+kernel+lemote: bump REL to remove RC markers

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>

KexyBiscuit committed:
linux-kernel-lemote: update to 5.4.285

- NO routine patch due diligence for lemote flavor releases as of now.
- Track patches at https://github.com/AOSC-Tracking/linux @ aosc/lemote/v5.4.285, current HEAD is d24618de8c547f05d3633d56e299b621181ab5fc.

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>

KexyBiscuit committed:
linux-kernel-lemote: fix SRCS URL

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>

MingcongBai commented:
The Lemote kernel is dead, long live Lemote.
